python/stressng-plugin/requirements.txt

```diff
@@ -0,0 +1,3 @@
+git+https://github.com/arcalot/arcaflow-plugin-sdk-python@main
```

Comment:
You should be able to change this now to `arcaflow-plugin-sdk` to get the SDK directly via pip.

Reply:
done